etc/HIReplayOfflineConfiguration.py

```diff
@@ -111,25 +107,25 @@
 
 # Defaults for CMSSW version
 defaultCMSSWVersion = {
-    'default': "CMSSW_14_1_3"
+    'default': "CMSSW_14_1_4_patch4"
```

Comment:
```suggestion
    'default': "CMSSW_14_1_4_patch5"
```
given the outcome of https://github.com/dmwm/T0/pull/5017.